AsyncNats/NatsConnection.cs

```diff
@@ -429,15 +477,49 @@ public async ValueTask PublishAsync(NatsKey subject, NatsPayload? payload = null
             await WriteAsync(pub, cancellationToken);
         }
 
-#if !DISABLE_PUBLISH_RAW
-        public async ValueTask PublishRaw(ReadOnlyMemory<byte> rawData,int messageCount, CancellationToken cancellationToken)
+        public async ValueTask PublishAsync(NatsKey subject, CancellationToken cancellationToken = default)
```

Comment:
The reason I removed these from the main branch is because of the parameter positioning confusion that can happen due to implicit conversions and the randomness of the parameter selection. string -> NatsKey is implicit, but so is string -> NatsPayload. It also adds a lot of code-overhead (a lot of nearly duplicate method bodies).

Reply:
You were right to do so. I was working on a stale branch and these overloads shouldn't be here indeed.